AEPMessaging/Tests/UnitTests/Message+FullscreenMessageDelegateTests.swift

```diff
@@ -103,4 +104,15 @@ class MessageFullscreenMessageDelegateTests: XCTestCase {
     func testOnShowFailureCallable() throws {
         message.onShowFailure()
     }
+    
+    func testOverrideUrlLoadAnimationOverride() throws {
+        // test
+        let result = message.overrideUrlLoad(message: mockFullscreenMessage, url: animationOverrideUrlString)
+        
+        // verify
+        XCTAssertFalse(result)
+        XCTAssertNil(mockMessage.paramTrackInteraction)
+        XCTAssertEqual(.left, mockMessage.fullscreenMessage?.settings?.dismissAnimation)
+        XCTAssertTrue(mockMessage.dismissCalled)
+    }
```

Comment:
would it be worth it to add an error test case for animation override? maybe something like `animation=` and verifying that the dismiss animation is `none`.

Reply:
@rymorale good call out. 

the existing behavior was more confusing than this...an empty value for `animation` was ignored, and so the original value was being used (in this case, it was `nil`).  i changed the behavior to not check for empty string, and always use `.none` as the animation type if there isn't a match found.

tests added for both empty and non-matching use cases